gravitee-am-common/src/main/java/io/gravitee/am/common/event/AlertNotifierEvent.java

```diff
@@ -26,18 +26,11 @@ public enum AlertNotifierEvent {
     UNDEPLOY;
 
     public static AlertNotifierEvent actionOf(Action action) {
-        AlertNotifierEvent notifier = null;
-        switch (action) {
-            case CREATE:
-                notifier = AlertNotifierEvent.DEPLOY;
-                break;
-            case UPDATE:
-                notifier = AlertNotifierEvent.UPDATE;
-                break;
-            case DELETE:
-                notifier = AlertNotifierEvent.UNDEPLOY;
-                break;
-        }
-        return notifier;
+        return switch (action) {
+            case CREATE -> AlertNotifierEvent.DEPLOY;
```

Comment:
Shouldn't `BULK_*` actions be mapped to the same `AlertNotifierEvents` as non-bulk ones?
(Same applies to other `Action` -> enum mappers)